### PR TITLE
Update to latest kimai 0.8.1 + adding Czech language to manifest.json

### PR DESCRIPTION
**Pull request by blueowl04**
Hello anmol26s,

please merge the following commits so that kimai is up-to-date for YunoHost users.

The changes update to latest Kimai release 0.8.1 and fixes a few problems.
There is also a commit adding Czech language to manifest questions.

Thanks & regards,
blueowl04

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -77,6 +77,9 @@ ynh_install_php7
 
 # Download, check integrity, uncompress and patch the source from app.src and do required installation for update
 
+# version before upgrade
+old_ver=`sudo yunohost app info $app --output-as plain | ynh_get_plain_key version`
```
Comment (anmol26s) [inline scripts/upgrade]: I took different approach for taking out version number from the core app file. But your way is much better and appropriate to yunohost way.
So I request you to remove the code on line 84 and keep your code at 81 as default way to get version number.

**Comment by anmol26s** [comment]
looks ok for me. Thanks to take care of the package in my absence. 
If @kevinpapst has plan to release the next version we can remove the https://github.com/YunoHost-Apps/kimai2_ynh/pull/14/commits/07ab5640a63fcbf3f0ac1b2e5df745efff613467 commit in this PR itself.

**Comment by kevinpapst** [comment]
Current master is not releasable and I don't have the time for a release right now.
I'd day: create a follow up issue for this task, so it can be removed with 0.9.

**Comment by anmol26s** [comment]
Great. Then we can go with this PR after version redundancy is removed.

**Comment by anmol26s** [comment]
Merging this in testing.

**Comment by blueowl04** [comment]
@anmol26s Thanks for merging my PR.
BTW, when do you plan to merge to master?

**Comment by anmol26s** [comment]
@blueowl04 
It has been merged.
Thank so much to show love to the package.

**Comment by kabums** [comment]
Hi,
I have some trouble with the customer part of the Installation. Can´t create new customers or change existing customer. 
I have also tried from a clean install of yunohost and kimai2. All customer related actions ends in..500: Internal Server Error.......
Please renew the App to kimai2 0,9 or fix the error (if possible)

Thanks!

**Pull request merged**